app/src/shared/components/MobileNav/Bar/bar.pcss

```diff
@@ -2,7 +2,7 @@
   background-color: white;
   display: flex;
   justify-content: space-between;
-  padding: calc(1.25 * var(--um)) calc(1.5 * var(--um));
+  padding: calc(1.25 * var(--um)) var(--um);
```

Comment:
Tightened the padding here to align the hamburger & close icon when toggling:
<img width="331" alt="Screen Shot 2020-03-05 at 17 20 03" src="https://user-images.githubusercontent.com/1064982/75996066-ff3ec380-5f05-11ea-9417-d11cd8ce77c6.png">